integration-tests/utils/HACBSApplications.ts

```diff
@@ -0,0 +1,96 @@
+import { applicationDetailPagePO } from '../support/pageObjects/createApplication-po';
+import { componentsTabPO } from '../support/pageObjects/hacbs-po';
+import { AddComponentPage } from '../support/pages/AddComponentPage';
+import { ComponentPage } from '../support/pages/ComponentsPage';
+import { AddIntegrationTestPage } from '../support/pages/hacbs/AddIntegrationTestPage';
+import { CreateBuildPage } from '../support/pages/hacbs/CreateBuildPage';
+import { Applications } from './Applications';
+import { Common } from './Common';
+
+export class HACBSApplications {
+    static deleteApplication(applicationName: string) {
+        Applications.deleteApplication(applicationName);
+    }
+
+    static createApplication(name: string) {
+        Applications.createApplication(name);
+    }
+
+    static createComponent(publicGitRepo: string, componentName: string) {
+        addComponentStep(publicGitRepo);
+        reviewComponentsStep(componentName);
+        createBuildStep();
+        addIntegrationTestStep();
+    }
+
+    static createdComponentExists(componentName: string, applicationName: string) {
+        this.goToComponentsTab();
+
+        Common.verifyPageTitle(applicationName);
+        Common.waitForLoad();
+        this.getComponentListItem(componentName).should('exist');
+    }
+
+    static getComponentListItem(application: string) {
+        return cy.contains(applicationDetailPagePO.item, application, { timeout: 60000 });
+    }
+
+    static goToComponentsTab() {
+        cy.get(componentsTabPO.clickTab).click({force: true});
+    }
+}
+
+function addComponentStep(publicGitRepo: string) {
+    const addComponent = new AddComponentPage();
+
+    // Enter git repo URL
+    addComponent.setSource(publicGitRepo);
+    // Check if the source is validated
+    addComponent.waitRepoValidated();
+    // Setup Git Options
+    addComponent.clickGitOptions();
+
+    //Next block commented out because of bug:
+    //https://issues.redhat.com/browse/HAC-1285
+
+    //addComponent.setGitReference(gitReference);
+    //addComponent.setContextDir(contextDir);
+    addComponent.clickNext();
+}
+
+
+function reviewComponentsStep(componentName: string) {
+    const componentPage = new ComponentPage();
+
+    // Edit component name
+    componentPage.editComponentName(componentName + "-temp");
+    cy.contains('div', componentName + "-temp").should('be.visible');
+
+    // Switch back to orginal name
+    componentPage.editComponentName(componentName);
+    cy.contains('div', componentName).should('be.visible');;
+
+    //Create Application
+    componentPage.createApplication();
+}
+
+
+function createBuildStep() {
+    const createBuildPage = new CreateBuildPage();
+
+    createBuildPage.clickNext();
```

Comment:
You can make it a one-liner by rewriting to `new CreateBuildPage().clickNext();`.